docs/ref/schema-editor.txt

```diff
@@ -81,11 +81,35 @@ Adds ``index`` to ``model``’s table.
 
 Removes ``index`` from ``model``’s table.
 
+``add_constraint()``
+--------------------
+
+.. method:: BaseDatabaseSchemaEditor.add_constraint(model, constraint)
+
+.. versionadded:: 2.2
+
+Adds ``constraint`` of types as specified in :doc:`constraints </ref/models/constraints>` to ``model``'s table.
+
+``remove_constraint()``
+-----------------------
+
+.. method:: BaseDatabaseSchemaEditor.remove_constraint(model, constraint)
+
+.. versionadded:: 2.2
+
+Removes ``constraint`` of types as specified in :doc:`constraints </ref/models/constraints>` from ``model``'s table.
```

Comment:
I would chop ``of types as specified in :doc:`constraints </ref/models/constraints>` `` because it is not totally true. We have also [`ExclusionConstraint`](https://docs.djangoproject.com/en/dev/ref/contrib/postgres/constraints/#exclusionconstraint) in `contrib.postgres` and 3rd party packages can define custom constraints (the same for `add_constraint()`).

Reply:
Fixed!